products/value.js

```diff
@@ -0,0 +1,14 @@
+const choose_pcs_id = document.getElementById("choose-pcs-id");
+const value = choose_pcs_id.value;
+const set_val = document.getElementById("output-val");
+choose_pcs_id.addEventListener('change', (e) => {
```

Comment:
maybe we should try to separate data and display.
and all display should be dynamic.

My image is like this.
// data
[{name, price}, {name, price}, {name, price}, {name, price}, {name, price}]

// display
document.innerHTML = `¥${data.price} (about ${data.price / something} per price)`

but I quit my task so soon, and i appreciate your effort!!
so it's just an offer, if you like it, pls try...

Reply:
Thank you for your response! I'll change it like what you say.